spec/spec_helper.rb

```diff
@@ -22,12 +22,33 @@
 
 Magma.instance.configure(YAML.load(File.read('config.yml')))
 
+Magma.instance.setup_db
+
+def truncate_attributes_table_on_error(&block)
+  yield
```

Comment:
Can we assign DatabaseCleaner to do this for us?

Reply:
Maybe? Since we're loading the data before DatabaseCleaner is setup, we'd have to configure and start DatabaseCleaner earlier. I'm not sure it's worth the trouble given we're only truncating a single table.